lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralLinearModelAlgorithm.cxx

```diff
@@ -431,8 +436,13 @@ void GeneralLinearModelAlgorithm::setCovarianceModel(const CovarianceModel & cov
   const UnsignedInteger optimizationDimension = reducedCovarianceModel_.getParameter().getSize();
   if (optimizationDimension > 0)
   {
-    const Point lowerBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationLowerBound"));
-    const Point upperBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationUpperBound"));
+    Point lowerBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationLowerBound"));
+    Point upperBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationUpperBound"));
+    if (normalize_)
```

Comment:
@dbarbier Be careful! Only the bounds corresponding to scale parameters should be transformed! They are at locations 0,...,inputTransformation().getInputDimension()-1. BTW, using the same bounds for all the parameters may be dangerous. A more robust default choice should be done, in particular for amplitude parameters, but it can be dne in another PR. A last remark: when the optimal amplitude is deduced from the other optimal parameters, its value is not checked against the associated bound, which is a good thing for the algorithm but rather strange from an optimization point of view.

Reply:
Okay, done